### PR TITLE
Tweaked snapshot params

### PR DESCRIPTION
**Pull request by arkpar**

### CODE REVIEW

**Comment by 5chdn** [comment]
```
2017-08-21 13:53:06  Encountered error during block restoration: Snapshot contained too many blocks. Expected 5000, got 4960
```

Getting this, warning sounds super weird.

File: ethcore/src/ethereum/ethash.rs [hunk 1]
```diff
@@ -39,7 +39,10 @@ pub const PARITY_GAS_LIMIT_DETERMINANT: U256 = U256([37, 0, 0, 0]);
 
 /// Number of blocks in an ethash snapshot.
 // make dependent on difficulty incrment divisor?
-const SNAPSHOT_BLOCKS: u64 = 30000;
+const SNAPSHOT_BLOCKS: u64 = 5000;
+/// Maximum number of blocks allowed in an ethash snapshot.
+const MAX_SNAPSHOT_BLOCKS: u64 = 300000;
```
Comment (rphmeier) [inline ethcore/src/ethereum/ethash.rs]: is it meant to be 30k or 300k?

Comment (arkpar) [inline ethcore/src/ethereum/ethash.rs]: good catch

**Comment by debris** [comment]
closes #6297

**Pull request merged**